src/objects/GetStartedChecklistSection__mdt.object

```diff
@@ -1,6 +1,32 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <CustomObject xmlns="http://soap.sforce.com/2006/04/metadata">
     <description>NPSP Get Started Page sections</description>
+    <fields>
+        <fullName>Belongs_to__c</fullName>
```

Comment:
How about the following?
- Label: "Page Type"
- Api Name: Page_Type__c
- Help Text: The type of page (Admin or End User) that the options in this GetStarted Section will be rendered in.
- Description: (same as help text)

What do you think @smarovich?

Reply:
@force2b How about "The type of Get Started page (Admin or End User) where these sections will appear."?